src/events.js

```diff
@@ -121,6 +114,17 @@ function dispatchEvent(handler, element, event) {
   callback(handler, [element.$context, event]);
 }
 
+function getItems(state, event, options) {
+  if (options.interaction.mode === 'point') {
+    return state.visibleElements.filter((element) => element.inRange(event.x, event.y));
+  }
+  const element = getNearestItem(state.visibleElements, event);
```

Comment:
if removed the `[0]` from this line from getNearestItems:
> .slice(0, 1)[0]; // return only the top item

then could just return the result of `getNearestItem`

Reply:
Ok
Honestly I left it as I found it.
Anyway it is a working in progress because I would like to implement all options pf interaction and then this function should be changed however 
What do you think?

Reply:
@kurkle I had a look and I was starting to add other modes, axis and intersect options. Nevertheless for the other modes it could be a little complex without the normalization of element properties (I mean x, y, x2, y2, width, height, centerX and centerY, as planned with PR #532).
That said, I would propose to stay with this kind of first implementation of interaction, only with 'point' and 'nearest', ignoring axis and intersect.
When PR #532 will be implemented, we could create another PR adding all missing modes and options.
make sense?

Reply:
> if removed the `[0]` from this line from getNearestItems:
> 
> > .slice(0, 1)[0]; // return only the top item
> 
> then could just return the result of `getNearestItem`

DONE

Reply:
> @kurkle I had a look and I was starting to add other modes, axis and intersect options. Nevertheless for the other modes it could be a little complex without the normalization of element properties (I mean x, y, x2, y2, width, height, centerX and centerY, as planned with PR #532).
> That said, I would propose to stay with this kind of first implementation of interaction, only with 'point' and 'nearest', ignoring axis and intersect.
> When PR #532 will be implemented, we could create another PR adding all missing modes and options.
> make sense?

or I can add inYRange and inXRange in all annotations

Reply:
Maybe the normalization should be done first?

Reply:
good question... maybe for this PR is not strictly needed. I see the normalization could help the other PR, #655.
I would suggest to go with a single method for inRange (as suggested below).